packages/world/src/systems/MineSystem.sol

```diff
@@ -346,16 +346,14 @@ library MineLib {
     notify(sleepingPlayerId, DeathNotification({ deathCoord: bedCoord }));
   }
 
-  function _requireReachable(EntityId caller, Vec3 coord) public view {
+  function _requireReachable(Vec3 coord) public view {
     Vec3[6] memory neighbors = coord.neighbors6();
     for (uint256 i = 0; i < neighbors.length; i++) {
       Vec3 neighbor = neighbors[i];
       ObjectType objectType = EntityUtils.getObjectTypeAt(neighbor);
-      EntityId entity = EntityUtils.getMovableEntityAt(neighbor);
-      bool isEmpty = objectType.isNonSolid() && (entity == caller || !entity._exists());
 
-      // If the neighbor is empty, we consider the coordinate reachable
-      if (isEmpty) return;
+      // If the neighbor is passthrough, we consider the coordinate reachable
+      if (objectType.isPassThrough()) return;
```

Comment:
hmm but players arent pass through right? so then I cant mine a block below someone with this?

Reply:
ohh actually the object type would be some passthrough, not player lol